### ISSUE TITLE
Question Regarding GIM-DKM Training

### ISSUE THREAD

**Issue by gyj155** [xuelunshen/gim#37]
First of all, thank you for your excellent work on this repository. I have some questions regarding the training process of GIM-dkm, specifically with respect to the dataset and loss functions.

In the original DKM model, the training was divided into indoor and outdoor models. For GIM, I understand that it was trained using data from the indoor model combined with YouTube data. Could you confirm if this is correct? Specifically, does the GIM training use a combination of MegaDepth, ScanNet, and YouTube datasets?

In the original model, depth loss and confidence loss were calculated based on the depth information from the dataset. However, when using YouTube videos, which have image pairs with matched ground truth, could you clarify what kind of loss function is used in this case for training?

I would greatly appreciate any clarification on these points.

**Comment by xuelunshen** [comment]
Thank you for your recognition.
1. GIM-dkm is trained from scratch using a combination of MegaDepth, ScanNet, and the YouTube datasets.
2. In training, video labels only calculate the depth loss (L2 Loss).

**Comment by gyj155** [comment]
Thank you for your reply!

**Comment by xuelunshen** [comment]
You're welcome, I will release the training code this month, you can check more details in the code.

**Comment by xuelunshen** [comment]
@gyj155 
Sorry for the long wait, I have now released the code for video data generation and training network (gim-loftr). Please refer to the [README](https://github.com/xuelunshen/gim?rgh-link-date=2024-12-21T14%3A44%3A53Z#%EF%B8%8F-video-preprocess) for relevant information.

The training code for gim-dkm will be released later.

**Comment by xuelunshen** [comment]
@gyj155 Thank you for your waiting, I have released the training code for $GIM_{DKM}$. See [README](https://github.com/xuelunshen/gim#%EF%B8%8F-training-network) for details.
